packages/liveblocks-react/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "target": "es2020",
     "module": "es2020",
     "lib": ["dom"],
+    "skipLibCheck": true,
```

Comment:
Was seeing some type errors inside `node_modules`, so I've disabled them here.

Reply:
Do you remember which ones?